lldb/source/Plugins/Language/Swift/SwiftCodeCompletion.cpp

```diff
@@ -253,8 +253,8 @@ SwiftCompleteCode(SwiftASTContext &SwiftCtx,
     // `completionCodeFile`, so that declarations that have been re-declared in
     // the user's current code take precedence over persistent declarations.
     DenseMap<Identifier, SmallVector<ValueDecl *, 1>> newDecls;
-    for (auto it = completionCodeFile.getTopLevelDecls().begin();
-         it != completionCodeFile.getTopLevelDecls().end(); it++)
+    auto topLevelDecls = completionCodeFile.getTopLevelDecls();
+    for (auto it = topLevelDecls.begin(); it != topLevelDecls.end(); it++)
```

Comment:
I think that we can write this better as:

```suggestion
    for (const auto decl : completionCodeFile.getTopLevelDecls())
```

Reply:
done